sips/sip-135.md

```diff
@@ -81,17 +81,31 @@ We're also adding support for the following:
 
 The ability to close the short position, utilizing the sUSD collateral locked to repay the excess debt. The mechanism is detailed in [SIP 122](https://sips.synthetix.io/sips/sip-122) for collapsible loans.
 
-- The effective exchange fee for exchanging `sUSD` to the borrowed debt (sETH, sBTC) will be charged for swapping the `sUSD` collateral to repay the loan.
+To summarize how this will work:
 
-- A base % fee to be charged and payable to the feepool for the function to repay using sUSD collateral instead of acquiring the borrowed synth to repay the borrowed synth.
+- Allow the borrower (user) to repay their short position with the sUSD collateral. 
+- Charge the effective exchange fee for sUSD -> sETH / sBTC   + a service fee that is configurable.
+- Burn the sUSD collateral and reduces the borrowed sETH / sBTC units by the amount repaid (less exchange fees).
 
-- Default base fee would be 15 bps (0.15%) on top of the exchange fee.
+The effective exchange fee for exchanging `sUSD` to the borrowed debt (sETH, sBTC) will be charged for swapping the `sUSD` collateral to repay the loan. A base % fee to be charged and payable to the feepool for the function to repay using sUSD collateral instead of acquiring the borrowed synth to repay the borrowed synth. Default base fee would be 15 bps (0.15%) on top of the exchange fee and as mentioned above the base fee is a configurable value.
 
-**Liquidations using sUSD collateral**
+### Liquidations using sUSD collateral
+
+This will allow anyone (the borrower or a liquidator bot) to partially liquidate a borrower's position using sUSD collateral when it is below the liquidation ratio. This is described in https://sips.synthetix.io/sips/sip-122. The Liquidator wouldn't need to exchange into sETH / sBTC (the borrowed asset) in order to liquidate. 
+
+To summarize how this will work:
+
+- The Liquidator can use the sUSD collateral to partially liquidate a position. (As long as the position is > 110% overcollateralised which covers the liquidation penalty of 10%).
+- Burn the sUSD collateral to repay the borrowed asset (that is shorted).
+- The 10% liquidation penalty is shared between the liquidator (75%) and and the fee pool for stakers (25%). 
+
+In terms of priorities, liquidateFromCollateral is less important than repayWithCollateral. We view liquidateFromCollateral as being a nice to have and thus are treating it as an optional feature to roll out with shorts on L2.
+
+In order to implement this feature:
 
 - Update API for `Collateral.sol` to implement `function liquidateFromCollateral(address borrower, uint id)` which collapses the loan using the collateral to fix the loan back to the minimum loan c-ratio.
 
-- Deprecate liquidations of shorts using the shorted synth as the repayment token in favor of liquidation using the sUSD collateral.
+- Deprecate liquidations of shorts using the shorted synth as the repayment token in favor of liquidation using the sUSD collateral. 
```

Comment:
please add remove whitespace in your IDE settings so these kinds of diffs aren't generated.